packages/firefuel/lib/src/order_by.dart

```diff
@@ -15,11 +15,20 @@ import 'package:equatable/equatable.dart';
 class OrderBy extends Equatable {
   final String field;
   final OrderDirection direction;
+  final bool byId;
 
   OrderBy({
     required this.field,
     OrderDirection direction = OrderDirection.asc,
-  }) : this.direction = direction.toAscDesc;
+  })  : this.direction = direction.toAscDesc,
+        this.byId = false;
+
+  /// Creates a condition to order your collection by document id
+  OrderBy.docId([
+    OrderDirection direction = OrderDirection.asc,
+  ])  : this.field = '__id__',
```

Comment:
opinion: I usually like to name fields that don't mean anything as 'unused' or 'fake' or something so other devs running across this don't think that `__id__` is something that needs to be there for this to work.

Feel free to keep it like this, but I think there's value in making sure others know this isn't required by Firebase